lib/blumquist.rb

```diff
@@ -51,6 +55,13 @@ def resolve_json_pointers
     end
   end
 
+  def resolve_json_pointer(type_def)
```

Comment:
`resolve_json_pointer!` [modifies](https://github.com/moviepilot/blumquist/pull/17/files#diff-cb52284b877d325638c7b0cefe622506R67) the schema of the current blumquist, in a way that you can't know what type the object is, that's why I created `resolve_json_pointer!`, which resolves the pointer without modifying the current schema